testsuite/tests/conftest.py

```diff
@@ -797,7 +804,7 @@ def _custom_tenant(name="t", autoclean=True, wait=True):
 
             unprivileged_client = client.ThreeScaleClient(admin_base_url, access_token, ssl_verify=False)
 
-            @backoff.on_exception(backoff.fibo, threescale_api.errors.ApiClientError, max_tries=8, jitter=None)
+            @backoff.on_exception(backoff.fibo, errors.ApiClientError, max_tries=8, jitter=None)
```

Comment:
This looks like unrelated and unnecessary change

Reply:
I will submit this in a different PR